build.gradle

```diff
@@ -63,8 +63,8 @@ ext {
     compileSdkVersion = 29
     minSdkVersion = 16
     targetSdkVersion = 29
-    versionCode = 18
+    versionCode = 19
 
     buildToolsVersion = "29.0.2"
-    versionName = "3.1.2"
+    versionName = "3.1.4-FM"
```

Comment:
intentional?

Reply:
Not intentional @nathanstasin. 
Thanks for the catch.